core/components/molecules/stepper/__stories__/variants/StepperWithSkip.story.jsx

```diff
@@ -58,7 +56,14 @@ export const stepperWithSkip = () => {
   );
 };
 
-const customCode = `() => {
+const customCode = `/*
+// style.css
+.Stepper-body {
+    height: 200px;
+}
```

Comment:
Instead of using fixed height please use `pt-11` classname on Button wrapper div